src/main/webapp/app/exercises/programming/shared/code-editor/file-browser/code-editor-file-browser-folder.component.html

```diff
@@ -21,11 +21,11 @@
         <button [disabled]="disableActions" (click)="setCreatingNodeInFolder($event, FileType.FOLDER)" class="btn btn-small">
             <fa-icon [icon]="faFolder" title="{{ 'artemisApp.editor.fileBrowser.createFolder' | artemisTranslate }}"></fa-icon>
         </button>
-        <button [disabled]="disableActions || (isCompressed && item.children && item.children.length)" (click)="setRenamingNode($event)" class="btn btn-small">
+        <button [disabled]="disableActions || (isCompressed && item.value.includes('/'))" (click)="setRenamingNode($event)" class="btn btn-small">
```

Comment:
Tested on ts1: I can not rename a compressed folder that does not contain any elements

Reply:
Hi, this is expected behaviour, since we do not (yet?) support renaming compressed folders.

I also tested it on develop. Although renaming compressed files is not disabled on develop, I get an error when I try to rename compressed folders: "File operation could not be executed. Please try again later."

Image showing error on develop when trying to rename compressed folders
![Bildschirmfoto 2023-07-26 um 11 19 30](https://github.com/ls1intum/Artemis/assets/47261058/8a810ee2-6983-422c-84ca-9f63a6ddbeee)